helix-core/src/test/java/org/apache/helix/monitoring/mbeans/TestDropResourceMetricsReset.java

```diff
@@ -144,9 +148,11 @@ public void testDropWithNoCurrentState() throws Exception {
 
     // stop the participant, so the resource does not exist in any current states.
     participant.syncStop();
+    LOG.warn("Stopped participant node sync: " + _unregisterLatch);
```

Comment:
This message is incorrect. There's no participant sync happening here. `syncStop()` means "synchronous stop". Let's revise the logs with the correct understanding of what's going on here.